dist/info.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Info</title>
+    <link rel="stylesheet" href="/style.css" />
```

Comment:
i think we don't need to connect here `style.css`
we need 3 css files:
`base.css` (base styles and rules, not specific for pages, footer, head, fonts etc)
`root.css` (for index.html)
`info.css` (for info.css)

and for each page you should include `base.css` and root or info